tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -107,6 +107,15 @@ setup() {
     pod_config=$(mktemp "${BATS_FILE_TMPDIR}/$(basename "${pod_config_template}").XXX")
     IMAGE="$image_pulled_time_less_than_default_time" NODE_NAME="$node" envsubst < "$pod_config_template" > "$pod_config"
 
+
+    # Set CreateContainerRequest timeout for qemu-coco-dev
+    if [ "${KATA_HYPERVISOR}" == "qemu-coco-dev" ]; then
+        create_container_timeout=300
```

Comment:
@BbolroC If I remember correctly, the default CreateContainer request timeout in the kubelet configuration is 120s. Did you change this value on s390x?

Reply:
Yes, it is configured on a cluster creation. Thanks!

Reply:
The point is that that max `120s` was not enough to proceed the tests.

Reply:
> The point is that that max `120s` was not enough to proceed the tests.

Got it.